### PR TITLE
tests for ElasticSearch FormES

### PR DESCRIPTION
**Pull request by czue**
probably easier to review all at once - there was a fair amount of churn in the tests

note that these tests won't actually run on travis because I haven't added the app to settings.py. this is because we don't actually install/run the right version of elastic on travis yet.

@benrudolph @snopoke. @esoergel and @NoahCarnahan might also be interested

### CODE REVIEW

File: testapps/test_elasticsearch/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+
+
```
Comment (dimagimon) [inline testapps/test_elasticsearch/__init__.py]: W391 blank line at end of file

File: corehq/form_processor/tests/utils.py [hunk 2]
```diff
@@ -211,9 +210,16 @@ def create_form_for_test(domain, case_id=None, attachments=None, save=True):
 </data>"""
 
 
-def get_simple_form_data(form_id, case_id=None):
+class TestFormMetadata(jsonobject.JsonObject):
+    domain = jsonobject.StringProperty(required=False)
+    user_id = jsonobject.StringProperty(default='cruella_deville')
+    time_end = jsonobject.DateTimeProperty(default=datetime(2013, 4, 19, 16, 53, 2))
```
Comment (snopoke) [inline corehq/form_processor/tests/utils.py]: ncie

**Comment by snopoke** [comment]
:+1:

File: corehq/tests/app_test_db_dependencies.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@
 # This means that replacing INSTALLED_APPS with the list of declared dependencies plus the app
 # should still allow `./manage.py test app` to pass.
 pillowtop: []
+testapps.test_elasticsearch: []
```
Comment (benrudolph) [inline corehq/tests/app_test_db_dependencies.yml]: :joy:

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**